tests/Message.hs

```diff
@@ -47,6 +47,10 @@ tests = testGroup "message tests"
   [ testProperty "message round trip" prop_messageRoundTrip
   , localOption (HedgehogTestLimit (Just 10000)) $
       testProperty "message round trip with From header" prop_messageFromRoundTrip
+  , testProperty "mailbox with consecutive spaces" $
+      -- https://github.com/purebred-mua/purebred-email/issues/56
```

Comment:
Nice!

Reply:
Yeah I think example-based test for a rarely-found counterexample are a good idea.